Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { baseURL } from '../../Constants/Config'
+// Universal fetch request using axios
+export default function universalFetch(setResponse, endpoint, onError, onSuccess) {
+    console.log('Fetch started');
+    setResponse({
+        data: null,
+        loading: true,
+        error: null,
+    });
+    axios
+        .post(baseURL + endpoint)
+        .then((resp) => {
```

Comment:
@HerlinRijo POST requests take in 2 parameters right?

Reply:
changed